### ISSUE TITLE
[API Feature Request]  Add createQuicklink method

### ISSUE THREAD

**Issue by gdsmith** [raycast/extensions#175]
```ts
createQuicklink(name: string, destination: string, openWith: appLike|undefined);
```

**Describe the feature and the current behavior/state.**

Allow developers to create properly formatted quicklinks from the api. Users could then choose to promote import links to the root search without having to manually configure them.

Examples:

Integrating `createQuicklink` with Slack's api and protocol urls would allow you to create quicklinks for your most used channels:

```ts
createQuicklink(channelName, `slack://channel?team=${teamId}&id=${channelId}`, slack)
```

A file search extension could then allow a quicklink for a favourite folder:

```ts
createQuicklink(dir.name, dir.path, finder)

```

Or the same thing but with a custom app defined:

```ts
createQuicklink(dir.name, dir.path, customApp)

```

**Will this change the current api? How?**

No

**Who will benefit with this feature?**

Raycast users :)

**Any Other info.**

I believe this would expand on the api's capabilities in a powerful way that aligns with Raycast's principles, gives users control and allow 3rd party developers to have a sensible/safe way of populating root search items

**Comment by unnamedd** [comment]
I like the idea as well, and I see it doable asking the user's permission before the creation, otherwise, it might fall exactly in the point mentioned by @pernielsentikaer!

**Comment by mathieudutour** [comment]
Available in v1.29.0

**Comment by pernielsentikaer** [comment]
And the documentation is updated, check out [Action.CreateQuicklink](https://developers.raycast.com/api-reference/user-interface/actions#action.createquicklink)
